compiler/z/codegen/BinaryEvaluator.cpp

```diff
@@ -1045,7 +1052,8 @@ iDivRemGenericEvaluator(TR::Node * node, TR::CodeGenerator * cg, bool isDivision
    //    R1+1 -> Quotient
    //
    // Note:  D and DR require that R1=0 before executing the division
-   bool needCheck = (!node->isSimpleDivCheck()            &&
+   // TODO: Remove this Java-ism from OMR and push it into OpenJ9
```

Comment:
If this is covered by #3685 then why not mention the issue here rather than a TODO?

Reply:
Ah yes, the issue was opened after-the-fact. Totally forgot to go back and update the reference in the code. Fixed in 7e888b5